velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -421,20 +422,45 @@ std::optional<TimestampUnit> getTimestampUnit(
   return std::nullopt;
 }
 
+std::optional<std::string> getParquetDataPageVersion(
+    const Config& config,
+    const char* configKey) {
+  const auto version = config.get<std::string>(configKey);
+  VELOX_CHECK(
+      version == "PARQUET_1_0" || version == "PARQUET_2_0",
+      "Invalid Parquet version.");
+  return version.value();
+}
+
 } // namespace
 
 void WriterOptions::processSessionConfigs(const Config& config) {
   if (!parquetWriteTimestampUnit) {
     parquetWriteTimestampUnit =
         getTimestampUnit(config, kParquetSessionWriteTimestampUnit);
   }
+
+  auto parquetDataPageVersionString =
+      getParquetDataPageVersion(config, kParquetSessionDataPageVersion);
+
+  if (parquetDataPageVersionString == "PARQUET_1_0") {
+    parquetDataPageVersion = arrow::ParquetDataPageVersion::V1;
+  }
 }
 
 void WriterOptions::processHiveConnectorConfigs(const Config& config) {
   if (!parquetWriteTimestampUnit) {
     parquetWriteTimestampUnit =
         getTimestampUnit(config, kParquetHiveConnectorWriteTimestampUnit);
   }
+
+  // auto parquetDataPageVersionString =
```

Comment:
remove dead code?

Reply:
sorry, missed this through rebasing - removed.